src/signals/incident-management/selectors.js

```diff
@@ -28,9 +28,26 @@ export const makeSelectDistricts = createSelector([selectIncidentManagementDomai
     : null
 );
 
+export const selectFixtures = createSelector(
```

Comment:
Please keep the selectors naming convention here: `makeSelectFixtures`.

Reply:
Ah, yes. I'll make it consistent.
I find it a very strange way of naming, though. 😅